common/utils/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -74,6 +75,30 @@ public static void deleteQuietly(File file) {
     }
   }
 
+  /** Copy src to the target directory simply. File attribute times are not copied. */
+  public static void copyDirectory(File src, File dst) throws IOException {
```

Comment:
If `src` is a File, it seems that a file-to-file copy will occur regardless of whether `target` exists, and no error will be reported.

Reply:
Thank you, @LuciferYang . I addressed your comment.